docs/hints-for-use-with-docker.md

```diff
@@ -14,6 +14,12 @@ docker run \
 
 **Note:** The single forward slash `/` between the environment variable `$PWD` and the `:` is required for PowerShell compatibility, as PowerShell otherwise interprets `:` as part of the environment variable. 
 
+### Setting custom certificates to Docker image keystore
+
+It is possible to install custom certificates when building the image so that they are installed into various keystores.
+To do this, specify `--build-arg CA_FILES=<path>` when running `docker build`. 
+Note that this requires the folder or certificate file to be inside the docker build context (usually the directory where the build is run, i.e. probably the directory the Dockerfile resides in). Otherwise, the folders cannot be copied into the Dockerfile.
```

Comment:
Add a line break before the last sentence.
Also: I think you don't mean to say that the files cannot be copied into the Dockerfile, but into the docker image.